src/System.Linq.Expressions/tests/HelperTypes.cs

```diff
@@ -391,6 +395,68 @@ public static Type BoolEnumType
                 return _boolEnumType;
             }
         }
+        
+        public static Type FloatEnumType
```

Comment:
This is not currently used but snuck its way into the PR, the cheeky bugger. I can remove it if you want but when I attempt to add support for float/double enums, it'll come back